Tasks/AzureIoTEdgeV2/deployimage.ts

```diff
@@ -273,7 +273,11 @@ export async function run(telemetryEvent: TelemetryEvent) {
   for (let path of findPaths) {
     console.log(tl.loc('CheckValidJson', path));
     try {
+#if NODE16
```

Comment:
Shouldn't these files also be placed in _buildConfigs?

Reply:
As I understand, it is just an alternative way that works the same. We were using it in some recent migration PRs, for example, [here](https://github.com/microsoft/azure-pipelines-tasks/blob/3d2f595d821dd32e83b4c43b535ea037db021a39/Tasks/PackerBuildV1/src/utilities.ts#L45C11-L45C11).